src/tools/cephfs/cephfs-shell

```diff
@@ -1122,35 +1147,48 @@ sub-directories, files')
 
     du_parser = argparse.ArgumentParser(
         description='Disk Usage of a Directory')
-    du_parser.add_argument('dirs', type=str, action=path_to_bytes,
-                           help='Name of the directory.', nargs='?',
-                           default='.')
+    du_parser.add_argument('paths', type=str, action=path_to_bytes,
+                           help='Name of the directory.', nargs='*',
+                           default=['.'])
     du_parser.add_argument('-r', action='store_true',
                            help='Recursive Disk usage of all directories.')
 
     @with_argparser(du_parser)
     def do_du(self, args):
         """
-        Disk Usage of a Directory
+        Print disk usage of a given path(s).
         """
-        if args.dirs == b'':
-            args.dirs = cephfs.getcwd()
-        for dir_ in args.dirs:
+        def print_disk_usage(files_list):
+            if isinstance(files_list, str):
+                files_list = (files_list, )
+
+            for f in files_list:
+                try:
+                    f = os.path.normpath(f)
+                    if f[0] is '/':
+                        f = '.' + f
+
+                    st = cephfs.lstat(f)
+
+                    if stat.S_ISDIR(st.st_mode):
+                        dusage = int(cephfs.getxattr(f,
+                                     'ceph.dir.rbytes').decode('utf-8'))
+                    elif stat.S_ISREG(st.st_mode) or stat.S_ISLNK(st.st_mode):
```

Comment:
@batrick 
Also, should `elif stat.S_ISREG(st.st_mode) or stat.S_ISLNK(st.st_mode):` be replaced by just `else:` here? Can any non-directory file not have `st.st_size`?